src/components/CharactersId.js

```diff
@@ -0,0 +1,5 @@
+function CharactersId() {
+  return <div></div>;
+}
+
+export default CharactersId;
```

Comment:
Can't see you using this component anywhere, but I assume it will end up being the individual character page

Reply:
You're correct! I've actually been working on this file right now.